### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by tobiaszheller**
This PR replaces the following PRs opened by dependabot:

- #17814 
- #17766
- #17765
- #17764
- #17763
- #17762
- #17761
- #17760
- #17759
- #17758
- #17757
- #17756
- #17755
- #17754
- #17753
- #17752
- #17751
- #17750
- #17749
- #17748

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -222,7 +222,7 @@ require (
 	github.com/go-openapi/swag v0.22.3 // indirect
 	github.com/gocql/gocql v1.2.1
 	github.com/golang-jwt/jwt v3.2.2+incompatible // indirect
-	github.com/golang-jwt/jwt/v4 v4.2.0
+	github.com/golang-jwt/jwt/v4 v4.4.2
```
Comment (codingllama) [inline go.mod:225]: Would you mind moving this out of the "// indirect" dependency block to the one above?

**Review by codingllama — COMMENTED** [review]
Anything noteworthy in release notes? github.com/golang-jwt/jwt/v4 seems like the biggest jump (4.2.x to 4.4.y).

**Review by nklaassen — COMMENTED** [review]
I should fix my script to print better links to the dependabot PRs, those api links aren't very nice lol

I should add a `go mod tidy` as well, I that's why lint and unit tests are failing on this right now

**Comment by codingllama** [comment]
@nklaassen if we could do something similar to "<title> #XXXX" it would be nice :)

**Comment by tobiaszheller** [comment]
> Anything noteworthy in release notes? github.com/golang-jwt/jwt/v4 seems like the biggest jump (4.2.x to 4.4.y).

Here are release notes: https://github.com/golang-jwt/jwt/releases
nothing catch my attention but maybe you could verify it also.

File: go.mod [hunk 2]
```diff
@@ -48,15 +48,16 @@ require (
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.10.0
 	github.com/gobuffalo/flect v0.3.0
+	github.com/gocql/gocql v1.2.1
```
Comment (tobiaszheller) [inline go.mod:51]: moved out of indirect

File: go.mod [hunk 3]
```diff
@@ -48,15 +48,16 @@ require (
...
 	github.com/gofrs/flock v0.8.1
 	github.com/gogo/protobuf v1.3.2 // replaced
+	github.com/golang-jwt/jwt/v4 v4.4.2
```
Comment (tobiaszheller) [inline go.mod:54]: moved out of indirect and bumped from v4.2.0 to v4.4.2

**Comment by tobiaszheller** [comment]
@codingllama @nklaassen PTAL, I have added better description and run go mod tidy. Tests are passing and nothing worth mentioning in release notes.

**Review by rosstimothy — APPROVED** [review]
LGTM. Don't forget to close the PRs opened by Dependabot in favor of this one.

File: go.mod [hunk 4]
```diff
@@ -48,15 +48,16 @@ require (
...
 	github.com/google/btree v1.1.2
 	github.com/google/go-cmp v0.5.9
 	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/websocket v1.5.0
 	github.com/gravitational/configure v0.0.0-20180808141939-c3428bd84c23
-	github.com/gravitational/form v0.0.0-20151109031454-c4048f792f70
```
[the remark this answers is not in the document]
Comment (tobiaszheller) [reply go.mod]: great 👁️ , thanks, added it to e_imports.go

**Review by codingllama — APPROVED** [review]

File: go.mod (hunk 4 above)
[the remark this answers is not in the document]
Comment (codingllama) [reply go.mod]: I broke the build like this a couple of times, so now I look at deleted lines more carefully.

**Pull request merged**